portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -366,6 +369,26 @@ public DLFileVersion cancelCheckOut(long userId, long fileEntryId)
 		return dlFileVersion;
 	}
 
+	@Override
+	public boolean canCopyFileEntry(
```

Comment:
canCopyFile Entry And can extract a method because is the same behavior than with folder